algo-builder-tester/assets/asa.yaml

```diff
@@ -1,4 +1,33 @@
-# minimum ASA definition (all optional fields omitted)
+$ART$:
+  total: 1e+12
+  decimals: 6
+  defaultFrozen: false
+  unitName: "ART"
+  url: "url"
+  metadataHash: "12345678901234567890123456789012"
+  note: "note"
+  noteb64: "noteb64"
+  # manager: "WWYNX3TKQYVEREVSW6QQP3SXSFOCE3SKUSEIVJ7YAGUPEACNI5UGI4DZCE"
```

Comment:
when these are not set, what are manager/reserve/freeze/clawback addr?

Reply:
they are "undefined" I assume they are set to creator. I'll do a test now

Reply:
> After an asset has been created only the manager, reserve, freeze and clawback accounts can be changed. All other parameters are locked for the life of the asset.
From [Algorand Standard Assets (ASAs)](https://developer.algorand.org/docs/get-details/asa/#modifying-an-asset)
So we cannot change the total number of coins. Maybe we publish more ART with less in circulation? I don't know how ALGO works.

Reply:
after test, I see they are set as "blank address" seems the creator can change all four accounts, if they are set to non-empty.

Reply:
test added: https://github.com/Artcoin-Network/artificial-algorand-contract/commit/adca6c30f1a471eb3896cfea7851b2a2c315ffcb